static/app/components/footer.tsx

```diff
@@ -107,7 +107,7 @@ const Footer = styled(BaseFooter)`
   color: ${p => p.theme.subText};
   border-top: 1px solid ${p => p.theme.border};
   padding: ${space(4)};
-  margin-top: 20px;
```

Comment:
Removing this means we can now delete all the negative margin hacks in other components